fail2ban/tests/utils.py

```diff
@@ -368,6 +448,38 @@ def assertNotIn(self, a, b, msg=None):
 
 class LogCaptureTestCase(unittest.TestCase):
 
+	class _MemHandler(logging.StreamHandler):
```

Comment:
Please provide the docstring describing its purpose and basic operation (not to delegate logging to StreamHandler at all, format lazily on demand in getvalue, etc)